sake/redis.py

```diff
@@ -691,6 +693,55 @@ async def set_user(self, user: users.User, /, *, expire_time: typing.Optional[Ex
         await client.set(int(user.id), self.marshaller.serialize_user(user), pexpire=expire_time)
 
 
+class PrefixCache(ResourceClient, traits.PrefixCache):
+    __slots__: typing.Sequence[str] = ()
+
+    @classmethod
+    def index(cls) -> typing.Sequence[ResourceIndex]:
+        # <<Inherited docstring from ResourceClient>>
+        return (ResourceIndex.PREFIX,)
+
+    def subscribe_listeners(self) -> None:
+        # <<Inherited docstring from sake.traits.Resource>>
+        super().subscribe_listeners()
+
+    def unsubscribe_listeners(self) -> None:
+        # <<Inherited docstring from sake.traits.Resource>>
+        super().unsubscribe_listeners()
+
+    async def clear_prefixes(self) -> None:
+        # <<Inherited docstring from sake.traits.PrefixCache>>
+        client = await self.get_connection(ResourceIndex.PREFIX)
+        await client.flushdb()
+
+    async def delete_prefixes(self, guild_id: snowflakes.Snowflakeish, /) -> None:
```

Comment:
When sset is used we'll probably want to have a `delete_prefix` or `delete_prefixes` method which targets specific prefix.

Reply:
Wouldn't it be delete_prefix(es)_for_guild to keep it in line with clear_prefixes_for_guild?

Reply:
`clear_prefixes_for_guild` is postfixed with `for_guild` to distinguish it from a global clear but when removing a specific entry from a guild bound resource it's just called `delete_{resource_name}` (similar to how it's handled for resources like members and presences) since there isn't going to be an equivalent global method